core/src/main/scala/com/databricks/labs/remorph/parsers/intermediate/commands.scala

```diff
@@ -87,3 +87,14 @@ case class WriteStreamOperationStart(
     foreach_batch: Option[StreamingForeachFunction])
     extends LeafNode
     with Command
+
+
+
+case class CreateVariable(name: LogicalPlan, defaultExpr: Expression, replace: Boolean) extends UnaryNode with Command {
+  override def child: LogicalPlan = name
+}
+
+case class SetVariable(name: String, value: LogicalPlan) extends UnaryNode with Command {
```

Comment:
```suggestion
case class SetVariable(name: String, value: Expression) extends UnaryNode with Command {
```
variables are results of expressions, not plans.

Reply:
I was making them as a command trying to map it to how Spark maps them.

Reply:
Either I keep them as command defined them as `leafnode` or create a new Branch called statements which raps `expressions`